eng/p-ops/2022-May-Timesheet-POPS-Part2.md

```diff
@@ -0,0 +1,13 @@
+# Summary
+* Owner: P-OPS Team
+* 2022 May 21 to 2022 May 31
+* Total: 4 hours 
+
+
+# Time Table
+| date  | start time  | end time | duration  |  note |
+|---|---|---|---|---|
+| 21-May-2022 | 11:30 | 12:30 | 1 | testnet user support (3) | 
+| 24-May-2022 | 09:00 | 09:30 | 0.5 | testnet user support (1) | 
+| 27-May-2022 | 12:15 | 13:15 | 1 | testnet user support (2) | 
+| 29-May-2022 | 17:15 | 18:45 | 1.5 | testnet user support (3) | 
```

Comment:
would you please add a bit more detail in the testnet user support?  like an id of the user being supported.

Reply:
Hello @LeoHChen I collected the list of users supported via our Telegram Support Channel. We will be adding the id's of either Telegram or Discord on next Pow's so it is more clear who is being supported. Here is the list os users:

https://t.me/A_chen518
https://t.me/Ardieean
https://t.me/omdur0
https://t.me/Primal1031
https://t.me/Dann_charles
https://t.me/ndagx
https://t.me/user98261
https://t.me/memomerp
https://t.me/efkah12